### PR TITLE
Create Letter-Combinations-of-a-Phone-Number.md

### PR DESCRIPTION
**Pull request by noelfoka**
Here we have proposed a solution on the Combinations of letters of a telephone number using the Javascipt

### CODE REVIEW

**Comment by dassiorleando** [comment]
@noelfoka Thank you for submitting the changes, only three points remain in my opinion.

1. The title should be changed since it still has the previous challenge title (copy and paste I guess).
image

2. It should be present in the **README** file as well so that we can click on it from the main algorithms page.
image

3. Let's use small letters for the file names.
image

I'm looking forward to the improvements, Thank you.

**Review by dassiorleando — CHANGES_REQUESTED** [review]
The three points I mentioned should be addressed.

**Comment by messaismael** [comment]
Allright We'll fix that

**Comment by dassiorleando** [comment]
Another one, the hit icon should match this new algorithm file name:
image

**Comment by messaismael** [comment]
@noelfoka please can you take a look at these mentioned points !
It's very important

**Comment by dassiorleando** [comment]
@messaismael, please considering the above mentioned points, could you come up with a small documentation located in the **algorithms** folder that shows how add a **new algorithm solution**?

You can call that file **guide.md**.

Here’s the associated issue: https://github.com/numerica-ideas/community/issues/34

**Comment by messaismael** [comment]
Sure @dassiorleando

File: algorithms/Letter-Combinations-of-a-Phone-Number.md
Comment (messaismael) [inline algorithms/Letter-Combinations-of-a-Phone-Number.md:1]: use lowercase letters to name file

`letter-combinations-of-a-phone-number.md`

**Pull request closed without merging**